modules/oe_subscriptions_anonymous/src/Form/AnonymousSubscribeForm.php

```diff
@@ -164,7 +169,14 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       return;
     }
 
-    $this->messenger()->addMessage($this->t('A confirmation e-email has been sent to your e-mail address.'));
+    $confirm_message = [
+      '#theme' => 'oe_subscriptions_anonymous_message',
+      '#title' => $this->t('A confirmation email has been sent to your email address'),
+      '#message' => $this->t('To confirm your subscription, <strong>please click on the confirmation link</strong> sent to your e-mail address.'),
```

Comment:
Instead of having the strings here, let's have everything in the template.

Reply:
Moved